modules/rosa-sts-aws-requirements.adoc

```diff
@@ -0,0 +1,38 @@
+[id="rosa-sts-customer-requirements_{context}"]
+= Customer Requirements
+
+
+{product-title} (ROSA) clusters must meet several prerequisites before they can be deployed.
+
+[id="rosa-account_{context}"]
+== Account
+* The customer ensures that the link:https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html[AWS limits] are sufficient to support {product-title} provisioned within the customer's AWS account.
+* If SCP policies are applied and enforced, these policies must not be more restrictive than the roles and policies required by the cluster.
+* The customer's AWS account should not be transferable to Red Hat.
+* The customer should not impose additional AWS usage restrictions beyond the defined roles and policies on Red Hat activities. Imposing restrictions will severely hinder Red Hat's ability to respond to incidents.
+* The customer may deploy native AWS services within the same AWS account.
```

Comment:
s/ may / can

Reply:
This language is also in the non-STS version.

Reply:
Language is the same in non-STS version.  Let's make that change later at the same time for both consistency and chance for proper review by product mgmt.